Assets/MixedRealityToolkit.Services/InputSystem/Editor/MixedRealityCanvasInspector.cs

```diff
@@ -0,0 +1,342 @@
+            {
+                if (!isMrtkCanvas)
+                {
+                    EditorGUILayout.PropertyField(m_Camera, Styles.eventCamera);
+
+                    if (m_Camera.objectReferenceValue == null)
+                        EditorGUILayout.HelpBox("A World Space Canvas with no specified Event Camera may not register UI events correctly.",
+                            MessageType.Warning);
+
+                    EditorGUILayout.Space();
+                }
+                sortingLayerField.Invoke(null, new object[] { Styles.m_SortingLayerStyle, m_SortingLayerID, EditorStyles.popup });
+                EditorGUILayout.PropertyField(m_SortingOrder, Styles.m_SortingOrderStyle);
+            }
+            EditorGUILayout.EndFadeGroup();
+            EditorGUI.indentLevel--;
+        }
+
+        private void AllNestedCanvases()
+        {
+            EditorGUI.BeginChangeCheck();
+            pixelPerfect = (PixelPerfect)EditorGUILayout.EnumPopup("Pixel Perfect", pixelPerfect);
+
+            if (EditorGUI.EndChangeCheck())
+            {
+                if (pixelPerfect == PixelPerfect.Inherit)
+                {
+                    m_PixelPerfectOverride.boolValue = false;
+                }
+                else if (pixelPerfect == PixelPerfect.Off)
+                {
+                    m_PixelPerfectOverride.boolValue = true;
+                    m_PixelPerfect.boolValue = false;
+                }
+                else
+                {
+                    m_PixelPerfectOverride.boolValue = true;
+                    m_PixelPerfect.boolValue = true;
+                }
+            }
+
+            EditorGUILayout.PropertyField(m_OverrideSorting);
+            m_SortingOverride.target = m_OverrideSorting.boolValue;
+
+            if (EditorGUILayout.BeginFadeGroup(m_SortingOverride.faded))
+            {
+                GUIContent sortingOrderStyle = null;
+                if (m_AllOverlay)
+                {
+                    sortingOrderStyle = Styles.sortingOrder;
+                }
+                else if (m_NoneOverlay)
+                {
+                    sortingOrderStyle = Styles.m_SortingOrderStyle;
+                    sortingLayerField.Invoke(null, new object[] { Styles.m_SortingLayerStyle, m_SortingLayerID, EditorStyles.popup });
+                }
+                if (sortingOrderStyle != null)
+                {
+                    EditorGUILayout.PropertyField(m_SortingOrder, sortingOrderStyle);
+                }
+            }
+            EditorGUILayout.EndFadeGroup();
+        }
+
+        private bool ShowMRTKButton()
+        {
+            Canvas canvas = (Canvas)target;
+
+            if (!canvas.rootCanvas)
+            {
+                return false;
+            }
+
+            bool isMRTKCanvas = canvas.GetComponent<CanvasUtility>() != null;
+
+            if (isMRTKCanvas)
+            {
+                if (GUILayout.Button("Revert MRTK canvas"))
+                {
```

Comment:
I would add some tooltip or info msg that says we're going to remove CanvasUtility and NearInteractionTouchableUnityUI